openqabot/amqp.py

```diff
@@ -104,5 +104,5 @@ def handle_incident(self, inc_nr: int, message: dict[str, Any]) -> None:
         approve = Approver(self.args, inc_nr)
         approve()
 
-    def handle_aggregate(self, unused_build: str, unused_message: dict[str, Any]) -> None:  # noqa: ARG002 Unused method argument
```

Comment:
`_` is a terrible name for an argument. Since the arguments are unused you're removing almost the last bit of information of what these arguments are meant to be. (Technically one can still check the call-sites but I prefer if a function is self-explaining.)

Maybe there's something like `[maybe_unused]]` in C++?

If not, you can also simply use these arguments, e.g. to print a log message.

Reply:
We can also keep the previous specific noqa exclusion comments in place. I still don't understand why we need to fill a method with unused arguments which does not do anything. Can I just delete the method and where it's called?

Reply:
You can also remove it if you like; it wasn't me who wrote this function. I think it was (and is) meant as a hint for future extension but if you don't like these kinds of structures in the code base we can remove it.

Reply:
ok, removed